ios/FluentUI/Resources/Localization/ar.lproj/Localizable.stringsdict

```diff
@@ -13,17 +36,17 @@
         <key>NSStringFormatValueTypeKey</key>
         <string>d</string>
         <key>zero</key>
-        <string>الساعة %d</string>
+        <string>%d o'clock</string>

```

Comment:
Not sure why the automated PR is reverting back to english for the stringsdict files. I can try pushing a commit to the tdbuild branch and reverting them, though.

Reply:
That seems to have cleaned it up.